tools/provision/centos.sh

```diff
@@ -28,4 +26,12 @@ function distro_main() {
 
   package rpm-devel
   package rpm-build
+
+  # CentOS 6 installs old ruby. gem install of fpm does not work with old ruby
```

Comment:
My preference is to never run `gem` on a CentOS 6 machine. You should not need to do this. The `osquery` you build on Ubuntu 16.04 will run on CentOS 6.

Reply:
@theopolis That sounds good. I will remove this code. I am guessing we don't need any ruby* packages on CentOS right? See the two lines I removed above.

BTW, I am kind'a curious about some of the "package *" in provision scripts. For eg where do we need unzip/bzip2 etc?

Reply:
These scripts are semi-legacy. The goal is to provide a `make sysprep` that gets each selected flavor up and running within Jenkins. And we can use the Vagrant hosts to mimic Jenkins. Some of these packages here were once used to bootstrap ALL of the dependencies, before there were `brew` formulas.

Reply:
Ok. Let me take a stab at cleaning some of these packages. That should reduce "make sysprep" time a little!

Reply:
I wonder if we could have a `make universe` that scripted starting each Vagrant, `make sysprep`, then `make` and `shutdown` 1 after the other. Sounds like a lot of work, but would be neat and provide confidence each of the package reductions would not cause provisioning regressions in the future. Jenkins will drop and re-provision hosts based on the need, but there's really no good way to support 0-> building within a PR right now.

Reply:
@theopolis We are kind'a doing this with Jenkins. A parent job forks bunch of jobs each of which run vagrant up box && vagrant ssh box -c "make sysprep && make && make test".. Shouldn't be that complicated. It will take a while though. We are doing this with few (4) slaves.